scalafix/build.sbt

```diff
@@ -27,29 +27,32 @@ lazy val rules = project.settings(
   libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafixVersion
 )
 
+val http4sInputVersion = "0.21.4"
+val http4sOutputVersion = "0.21.4"
```

Comment:
0.18 is ancient, but it's also a shame to lose the old rules.  Is there a way to version these, so people might run multiple iterations if catching up across multiple versions?  Probably related to these "semantic rules," but I would need to refresh my memory how these work.

Reply:
The problem is that the old rules don't even compile with the newer http4s version. I'd also prefer to keep the old rules but have no idea how to achieve it easily. But I'm open to any suggestions:)

Reply:
Perhaps @amarrella remembers how this works, as the original contributor.  If not, we can learn together. :smile:

Reply:
It's a long time since I've looked at these but can we perhaps set the old version as a dependency instead of depending on the project?

Reply:
@amarrella sorry, not sure if I caught it. Could you explain your idea a bit more please?

Reply:
@satorg apologies, I misunderstood the problem. I see the issue now with keeping `http4sInputVersion = 0.18.x` :( 
Perhaps the cleanest version is getting rid of the rules like you did, and point users to use a older version of the jar if they need the older rule and then updating to the new one. I know it's way less ergonomic though :/

Reply:
Well, I think we also could convert the **scalafix** directory into a multi-module project where each module has its own set of dependencies. But I'm not sure if it's a really good way to go.